### ISSUE TITLE
Restructuring navbar

### ISSUE THREAD

**Issue by jradha11** [The-Codesis/codesis-website#16]
Currently, the navbar consists of only links which are left-aligned. I think we should add our organizations' name and logo to the left and links should be floated to right.

**Comment by jradha11** [comment]
@NidhiKJha review required on the issue

**Comment by NidhiKJha** [comment]
Yes, I really liked your suggestion. But keep in mind that we will have to work for the mobile view as well. How are we going to manage all the fields in navbar the mobile view without decreasing font size in mobile view :)

**Comment by jradha11** [comment]
We can use a ham for mobile version

**Comment by Yogita98** [comment]
@jradha11 Is anyone currently working on this issue?

**Comment by jradha11** [comment]
@Yogita98 Not yet.

**Comment by Yogita98** [comment]
@jradha11 I would like to solve this issue.

**Comment by NidhiKJha** [comment]
Hi @jradha11  I am not sure about the mobile yet, we can think about it later. But I would really like it in the desktop mode. Thanks :)

**Comment by jradha11** [comment]
@NidhiKJha So should I rename the issue as Restructuring navbar for desktop mode?

**Comment by NidhiKJha** [comment]
No, Add changes for both modes. If your solution will be good we can add it otherwise I will help you to work on the mobile view :)

**Comment by jradha11** [comment]
@NidhiKJha Got it! Thank you :D
